GVFS/GVFS.Common/Prefetch/Git/DiffHelper.cs

```diff
@@ -24,18 +24,21 @@ public class DiffHelper
         private Enlistment enlistment;
         private GitProcess git;
 
-        public DiffHelper(ITracer tracer, Enlistment enlistment, IEnumerable<string> fileList, IEnumerable<string> folderList)
-            : this(tracer, enlistment, new GitProcess(enlistment), fileList, folderList)
+        private bool includeSymlinks;

```

Comment:
Same comment that Saeed left in a different file, if we've been using SymLink use that casing here as well